cinema/views.py

```diff
@@ -35,7 +37,7 @@ class GenreViewSet(
 ):
     queryset = Genre.objects.all()
     serializer_class = GenreSerializer
-    authentication_classes = (TokenAuthentication,)
+    authentication_classes = (JWTAuthentication,)
```

Comment:
You don't need to override authentication_classes, 'cause you have already added JWTAuthentication as default in settings.